### PR TITLE
Quarantine sg

### PR DESCRIPTION
**Pull request by ahayden**
Adds security group options

### CODE REVIEW

File: templates/vpc.yaml [hunk 1]
```diff
@@ -502,6 +502,31 @@ Resources:
           Value: !Ref Project
         - Key: "OwnerEmail"
           Value: !Ref OwnerEmail
+  QuarantineSecurityGroup:
+    Type: 'AWS::EC2::SecurityGroup'
+    Properties:
+      GroupDescription: Security Group for bastians
```
Comment (zaro0508) [inline templates/vpc.yaml]: is this accurate, shouldn't it be about quarentining?

File: templates/vpc.yaml [hunk 2]
```diff
@@ -502,6 +502,31 @@ Resources:
...
+      VpcId:
+        Ref: "VPC"
+      SecurityGroupIngress:
+        - CidrIp: "10.0.0.0/8"
```
Comment (zaro0508) [inline templates/vpc.yaml]: shouldn't this be ingress from our VPN?

File: templates/vpc.yaml (hunk 1 above)
Comment (ahayden) [reply templates/vpc.yaml]: yeah, updated

File: templates/vpc.yaml (hunk 2 above)
Comment (ahayden) [reply templates/vpc.yaml]: it's shorter this way... but i updated it with the VPN range instead

**Pull request merged**